features/plugin-update.feature

```diff
@@ -191,3 +191,19 @@ Feature: Update WordPress plugins
       """
       Error: Can't find the requested plugin's version 2.5.4 in the WordPress.org plugin repository (HTTP code 404).
       """
+
+  Scenario: Plugin updates that error should not report a success
```

Comment:
This currently breaks on WP versions below v4.7.0, as Akismet v4.1 requires the presence of `WP_REST_Controller`.

I suggest just skipping the test for WP < 4.7.0.

```suggestion
  @require-wp-4.7
  Scenario: Plugin updates that error should not report a success
```